mobile-core.addon

```diff
@@ -3,41 +3,30 @@
 # Var-Defaults: CONTAINER_REPO_USERNAME=USERNAME, CONTAINER_REPO_PASSWORD=PASSWORD, CONTAINER_REPO_ORG=aerogearcatalog, VER=v9.3.0, CORE_REPO=aerogear, CORE_BRANCH=master, CONSOLE_REPO=aerogear, CONSOLE_BRANCH=master
 # OpenShift-Version: >=3.9.0
 
+echo Installing oc client in vm
+echo
 ssh tce-load -wi bash.tcz
 ssh tce-load -wi coreutils.tcz
-
-ssh curl -s https://nodejs.org/dist/#{VER}/node-#{VER}-linux-x64.tar.gz > ~/#{VER}.tar.gz
-ssh cd /usr/local && sudo /usr/local/bin/tar --strip-components=1 -xzf ~/#{VER}.tar.gz
-
 ssh sudo chmod -R a+rwx /var/lib/minishift/openshift.local.config
 ssh curl -s -O -J -L https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
 ssh tar -xf openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz && sudo cp openshift-origin-client-tools-v3.9.0-191fece-linux-64bit/oc /usr/bin/oc
 ssh mkdir ~/.kube
 ssh cp /var/lib/minishift/openshift.local.config/master/admin.kubeconfig ~/.kube/config
 ssh oc login -u system:admin
 
+echo Installing mobile-client CRD: repo=#{CORE_REPO} branch=#{CORE_BRANCH}
+echo
 oc login -u system:admin
 oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated
-
 oc create -f https://raw.githubusercontent.com/#{CORE_REPO}/mobile-core/#{CORE_BRANCH}/installer/roles/create-mobile-client-crd/files/mobile-client-crd.yaml
-
 oc create clusterrole mobileclient-admin --verb=create,delete,get,list,update,watch --resource=mobileclients
 oc adm policy add-cluster-role-to-group mobileclient-admin system:authenticated
 
-ssh git clone https://github.com/#{CORE_REPO}/mobile-core.git
-ssh cd ~/mobile-core && git checkout #{CORE_BRANCH}
-ssh cp -r ~/mobile-core/ui/* /var/lib/minishift/openshift.local.config
-echo Installing and starting MCP extension, this can take a while
-echo
-ssh bash /var/lib/minishift/openshift.local.config/install.sh
-ssh cd /var/lib/minishift/openshift.local.config/ && npm run grunt build
-docker restart origin
-ssh while [ $(yes | oc login -u system:admin | wc -l) -eq 0 ]; do sleep 2; done
-
-echo Installing Ansible Service Broker with Container Repository Organisation: #{CONTAINER_REPO_ORG}
+echo Installing Ansible Service Broker: repo=#{CORE_REPO},branch=#{CORE_BRANCH},container registry organisation=#{CONTAINER_REPO_ORG},container registry username=#{CONTAINER_REPO_USERNAME}
 echo
-ssh bash ~/mobile-core/installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh '#{CONTAINER_REPO_USERNAME}' '#{CONTAINER_REPO_PASSWORD}' '#{CONTAINER_REPO_ORG}' true latest #{ip} nip.io ansible-service-broker
+ssh curl -s -O -J -L https://raw.githubusercontent.com/#{CORE_REPO}/mobile-core/#{CORE_BRANCH}~/installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh
```

Comment:
@philbrookes @maleck13  we may be able to simplify this a lot more (separate PR) if we leverage the broker addon https://github.com/minishift/minishift-addons/tree/master/add-ons/ansible-service-broker
Thoughts on doing that?